packages/v/vid.stab/abi_libs

```diff
@@ -1 +1 @@
-libvidstab.so.1.1
```

Comment:
The change to the soname is going to require rebuilds of all reverse dependencies. That appears to just be `mlt`, can you please add another commit with that package getting a release number bump and being rebuilt? If correct you should see `libvidstab.so.1.2` appear in the `abi_used_libs` file on that package. You set the commit message to something like `mlt: Rebuild against vid.stab`

Reply:
`mlt` rebuild output `libvidstab.so.1.2` on `abi_used_libs`. I did use `go-task build-local`, already copy `.eopkg` files to `/var/lib/solbuild/local` , but when it download builddeps it shows that it download repo version of `vid.stab` not the local one. Is that normal?